### DISCUSSION TITLE
mega menu with quasar

### DISCUSSION THREAD

**Discussion by anli-xsigns** [quasarframework/quasar#12693] [General - Components / Directives / etc]
Hello *,

I wonder if there is any mega menu like component for quasar? I googled around but didn't find anything. Does somebody have a link to a component or maybe a code snippet?

Thanks a lot for your help!

Best, André

**Comment by pikil** [comment]
There is no such specific component, but you can achieve it with `q-tabs` and `q-menu` in `q-toolbar`. That should do the trick :)
